### ISSUE TITLE
Add game limit

### ISSUE THREAD

**Issue by martinoj2009** [martinoj2009/JH7#4]
Going to have the person setting up the server set the goal limit.

**Comment by martinoj2009** [comment]
Right now the server is handling the game limit, not sure how I can send a message using gameNet to the client what the score limit is... Still testing. Then need to know what to do when the limit is meet. Right now I just have the server closing... haven't found how to reset the game unless I can send messages to tell the client to reset the score.

**Comment by martinoj2009** [comment]
Added reset message. When the score is hit the reset message tells the server and client to close until we figure out what to do. Having difficulties resetting the score to 0.

**Comment by mgebremedhin** [comment]
I could create a routine in the "Box" class that will have both player's scores effectively reset to '0' again. That way, the Box-class object in the "MyUserInterface" class can be implemented to call that reset method.

Let me know what your thoughts are on this.

**Comment by martinoj2009** [comment]
That sounds good, lets try that.

**Comment by mgebremedhin** [comment]
I'm having issues now loading another instance of Pong for Player 2 as the client. The following is the error message received:

markServerStarted
 getPortNum = 54321
0 Another Thread Created
1 Another Thread Created
GamePlayerProcess2.run Err: java.io.EOFException
GamePlayerProcess2.run terminating: 1
GamePlayerProcess1.run Terminating : 1

**Comment by mgebremedhin** [comment]
The latest commit I was able to pull allows me to initiate the server-side and client-side windows for as long as the duration of the first game. Afterwards, the game stalls when re-initiating a new game.

**Comment by mgebremedhin** [comment]
Now, I'm not even able to start both instances from the very beginning.

**Comment by martinoj2009** [comment]
Are you still not able to? I'm able to run a server and client.

**Comment by mgebremedhin** [comment]
No, I still can't. Do me a favor and push out our latest version so that I can pull it and work with it. I want to get the JING video created and uploaded before today ends.

**Comment by martinoj2009** [comment]
The latest version is out. I'm running it. What errors are you getting?

**Comment by mgebremedhin** [comment]
When I run MyMain.java for the first time, it works as intended when setting it up as the server and with a score limit of 5 points. Yet, when I run another instance of MyMain.java to create the client window (after choosing "no" for becoming the server, setting the IP address to "localhost" and setting "54321" as the port number), I get the following message in Eclipse:

Getting outputs before we are ready
Getting outputs before we are ready
Getting outputs before we are ready
Getting outputs before we are ready
Getting outputs before we are ready
Getting outputs before we are ready
Getting outputs before we are ready
Getting outputs before we are ready
Getting outputs before we are ready
Getting outputs before we are ready
Getting outputs before we are ready
Getting outputs before we are ready
Getting outputs before we are ready
Getting outputs before we are ready
Getting outputs before we are ready
Getting outputs before we are ready
Getting outputs before we are ready
Getting outputs before we are ready
Getting outputs before we are ready
Getting outputs before we are ready
Getting outputs before we are ready
Getting outputs before we are ready
Getting outputs before we are ready
Getting outputs before we are ready
Getting outputs before we are ready
Getting outputs before we are ready
Getting outputs before we are ready
Getting outputs before we are ready
Getting outputs before we are ready
Getting outputs before we are ready
Getting outputs before we are ready
Getting outputs before we are ready
Getting outputs before we are ready
Getting outputs before we are ready
Getting outputs before we are ready
Getting outputs before we are ready
Not allowed to connect to the game
GamePlayer.exitGame

 markServerStarted
Listening
 getPortNum = 54321
Starting GameControl Server ipAddress(172.22.9.141)  portNum (54321)
0 Another Thread Created
Listening
1 Another Thread Created
Listening
Player 2 disconnected.
Other player disconnected

**Comment by martinoj2009** [comment]
We can try reverting to a previous commit to see what commit did this. It's
so weird that's happening. Java is supposed to be cross-platform. I'm
writing on Linux, shouldn't be that different.

**Comment by martinoj2009** [comment]
Try stepping your commits back. I would, but it works for me, I don't know what will and won't work for you.
http://stackoverflow.com/questions/4114095/how-to-revert-git-repository-to-a-previous-commit

**Comment by mgebremedhin** [comment]
I'm using SourceTree on my Macbook as an interface between GitHub and Eclipse.

**Comment by martinoj2009** [comment]
Cool, have you had any luck?

**Comment by mgebremedhin** [comment]
Yeah, it did work. I've gone ahead and uploaded a JING presentation of the work to my JH7 submittal page. When do you plan on uploading it to yours?

**Comment by martinoj2009** [comment]
Dang, where have you been???

**Comment by mgebremedhin** [comment]
Hahaha. I’m outside of The States now, but I should be back at WCC on August 29th.

How’ve you been, man? How’s your summer since our Java course ended?
